.editorconfig

```diff
@@ -67,79 +67,79 @@ csharp_preserve_single_line_statements = true
 #Style - Code block preferences
 
 #prefer curly braces even for one line of code
-# csharp_prefer_braces = true:suggestion
+csharp_prefer_braces = true
 
 #Style - expression bodied member options
 
 #prefer block bodies for accessors
-# csharp_style_expression_bodied_accessors = false:suggestion
+csharp_style_expression_bodied_accessors = true
 #prefer block bodies for constructors
-# csharp_style_expression_bodied_constructors = false:suggestion
+csharp_style_expression_bodied_constructors = true
 #prefer block bodies for methods
-# csharp_style_expression_bodied_methods = false:suggestion
+csharp_style_expression_bodied_methods = true
 #prefer block bodies for properties
-# csharp_style_expression_bodied_properties = false:suggestion
+csharp_style_expression_bodied_properties = true
 
 #Style - expression level options
 
 #prefer out variables to be declared before the method call
-# csharp_style_inlined_variable_declaration = false:suggestion
+csharp_style_inlined_variable_declaration = false
 #prefer the language keyword for member access expressions, instead of the type name, for types that have a keyword to represent them
-# dotnet_style_predefined_type_for_member_access = true:suggestion
+dotnet_style_predefined_type_for_member_access = true
 
 #Style - Expression-level  preferences
 
 #prefer objects to be initialized using object initializers when possible
-# dotnet_style_object_initializer = true:suggestion
+dotnet_style_object_initializer = true
 #prefer inferred anonymous type member names
-# dotnet_style_prefer_inferred_anonymous_type_member_names = false:suggestion
+dotnet_style_prefer_inferred_anonymous_type_member_names = false
 #prefer inferred tuple element names
-# dotnet_style_prefer_inferred_tuple_names = true:suggestion
+dotnet_style_prefer_inferred_tuple_names = true
 
 #Style - implicit and explicit types
 
 #prefer var over explicit type in all cases, unless overridden by another code style rule
-# csharp_style_var_elsewhere = true:suggestion
+csharp_style_var_elsewhere = true
 #prefer var is used to declare variables with built-in system types such as int
-# csharp_style_var_for_built_in_types = true:suggestion
+csharp_style_var_for_built_in_types = true
 #prefer var when the type is already mentioned on the right-hand side of a declaration expression
-# csharp_style_var_when_type_is_apparent = true:suggestion
+csharp_style_var_when_type_is_apparent = true
```

Comment:
This is surprising to me that this doesn't cause formatting changes.  There's plenty of places that say "int variable" in the code.

Also, I think to me var is nice but not required.  I'm happy to make it required if that's what you'd like, but I think sometimes explicit types can be clearer to read.

Reply:
Personally I prefer explicit types except when it's some really nasty type with a bunch of generic type parameters or something like that.

Reply:
Sounds like the answer is to allow but not require `var` then.

I think this is really my only open question on this PR.  Maybe I just don't understand what this settings means, but I wanted to understand why this didn't cause formatting changes elsewhere, e.g. all of the spots listed via `git grep "^\s*int "`.